app/admin/authors.rb

```diff
@@ -0,0 +1,16 @@
+ActiveAdmin.register Author do
+  permit_params :first_name, :last_name, :biography
+  actions :index, :new, :create, :update, :edit, :destroy
```

Comment:
As I remember all actions available by default. You can set some specific actions `only: [:index, :show]`. Or you can exclude some actions `exclude: [:delete]`